helm/content-store/values.yaml

```diff
@@ -0,0 +1,46 @@
+# Default values for frontend.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# values used to compute other values
+govukEnvironment: test
+govukStack: pink
+govukDomainExternal: govuk.digital
+govukDomainInternal: govuk-internal.digital
+clusterDomain: svc.cluster.local
+
+service:
+  annotations: {}
+  type: NodePort
+  port: 80
+  targetPort: 80
+
+deployment:
+  replicaCount: 1
+  image:
+    repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/content-store
+    tag: latest
+  containerPort: 80
+  resources:
+    limits:
+    # cpu:
+    # memory: 1024M
+    requests:
+    # cpu:
+    # memory: 1024M
+  # values used to generate the environment variables for the pod
+  appDefaultTll: "1800"
+  appGovukAppDomain: #default (printf "%s.%s.%s" .Release.Namespace .Values.govukEnvironment .Values.clusterDomain )
```

Comment:
I sort of agree with you, it was a tradeoff because in theory, the user should see the default values in values.yaml directly.

Reply:
I think it's not a good tradeoff because of the risk that they'll go out of date and then you'll have misleading comments, which is much worse than no comments.

Reply:
ok, I will put a note to say look at the template for the computed default value.